lib/contentful-management.js

```diff
@@ -1,26 +0,0 @@
-/**
- * Contentful Management API SDK. Allows you to create instances of a client
- * with access to the Contentful Content Management API.
- * @packageDocumentation
- */
-
-import createContentfulApi from './create-contentful-api'
-import { createCMAHttpClient } from './create-cma-http-client'
-
-/**
- * Create a client instance
- * @param params - Client initialization parameters
```

Comment:
Intended that this went missing?

Reply:
This comment is not used in the documentation generated from sources but I can bring it back.

Reply:
Isn't it this one? https://contentful.github.io/contentful-management.js/contentful-management/5.26.5/globals.html#createclient

Reply:
👀